dev-infrastructure/templates/dev-acr.bicep

```diff
@@ -12,6 +12,21 @@ param acrSku string
 @description('Set to true to prevent resources from being pruned after 48 hours')
 param persist bool
 
+@description('Captures logged in users UID')
```

Comment:
I simply cargo culted this, cause az gave me warnings during deployment. Is this correct?